website/src/graphql/dailp/index.ts

```diff
@@ -1034,6 +1039,25 @@ export type NewPageMutation = { readonly __typename?: "Mutation" } & Pick<
   "updatePage"
 >
 
+export type DocSliceQueryVariables = Exact<{
+  slug: Scalars["String"]
+  startIndex: Scalars["Int"]
+  endIndex: InputMaybe<Scalars["Int"]>
```

Comment:
You need to run the front-end type generation since you renamed these parameters. You always need to when you change a query.